src/ansys/aedt/core/modules/material.py

```diff
@@ -2377,7 +2377,7 @@ def set_coreloss_at_frequency(
                 self._props["AttachedData"]["CoreLossMultiCurveData"]["AllCurves"]["OneCurve"].append(one_curve)
 
         coefficients = self.get_core_loss_coefficients(
-            points_at_frequency, thickness=thickness, conductivity=conductivity
+            points_at_frequency, thickness=thickness, conductivity=conductivity, coefficient_setup=coefficient_setup
```

Comment:
```suggestion
            points_at_frequency,
            core_loss_model_type=core_loss_model_type,
            thickness=thickness,
            conductivity=conductivity,
            coefficient_setup=coefficient_setup
```
Since you are adding `coefficient_setup`, I don't see a reason not to add `core_loss_model_type` at the same time.